### DISCUSSION TITLE
Creating a non-optimized build?

### DISCUSSION THREAD

**Discussion by laurent22** [facebook/docusaurus#9467] [Q&A]
Is there any flag that can be used to create a non-optimized (and hopefully faster) build? I see that the Babel config is not exposed, and although passing `--no-minify` helps a bit, it's still creating an optimized build.

Any idea how to do this?

**Comment by slorber** [comment]
The babel config is exposed and we have an issue also showing how to provide your own JS loader. Our website use swc for example, just copy our setup.

V3 provides a build --dev flag but it's not particularly supposed to make it faster.
